packages/nameguard-sdk/src/index.ts

```diff
@@ -66,7 +66,10 @@ export type CheckType =
 
   // Name-level checks
   | "impersonation_risk" /** A name might be used for impersonation. */
-  | "punycode_compatible_name" /** A name is compatible with Punycode. */;
+  | "punycode_compatible_name" /** A name is compatible with Punycode. */
+  | "namewrapper_fuses" /** The NameWrapper configuration of a name is safe. */
+  | "decentralized_name" /** A name is decentralized. */
+  | "uninspected" /** A name was exceptionally long and was not inspected for performance reasons */;
```

Comment:
@notrab Hey, we've added 3 new potential checks here. Can you please investigate and confirm if we need to update anything in `nameguard-react`, `examples.nameguard.io` or `nameguard.io`?